source/stylesheets/refills/unstyled/_navigation-centered.scss

```diff
@@ -0,0 +1,93 @@
+.navigation-centered {
+  $large-screen: em(860) !default;
+  $centered-navigation-height: 60px;
+  $centered-navigation-logo-height: 2em;
+
+  width: 100%;
+
+  // Mobile view
+  .mobile-logo {
+    display: inline;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets